### PR TITLE
Consistent undeploy cmd

### PR DESCRIPTION
**Pull request by tobiscr**
**Description**

Align the output and the support flags for the `alpha uninstall` (name not aligned yet) command with the existing `alpha deploy` command.

**Related issue(s)**
#692

### CODE REVIEW

File: internal/k3s/k3s.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	k3dMinVersion  string        = "3.4.0"
+	k3dMinVersion  string        = "4.0.0"
```
Comment (tobiscr) [inline internal/k3s/k3s.go:16]: @valentinvieriu : JFYI - this PR will introduce K3d 4.0.0 support

File: cmd/kyma/alpha/uninstall/cmd.go [hunk 2]
```diff
@@ -57,30 +64,63 @@ func (cmd *command) Run() error {
...
+		approvalRequired := !os.IsNotExist(err)
+
+		if err := deploy.CloneSources(stepFactory, cmd.opts.WorkspacePath, kymaMeta.Version); err != nil {
 			return err
 		}
-		defer asyncUI.Stop() // stop receiving update-events and wait until UI rendering is finished
+		// delete workspace folder
+		if approvalRequired {
+			userApprovalStep := stepFactory.AddStep("Workspace folder exists")
+			if userApprovalStep.PromptYesNo(fmt.Sprintf("Delete workspace folder '%s' after Kyma was removed?", cmd.opts.WorkspacePath)) {
```
Comment (suleymanakbas91) [inline cmd/kyma/alpha/uninstall/cmd.go:112]: We shouldn't prompt in case `non-interactive` or `ci` flags are passed.

**Review by suleymanakbas91 — CHANGES_REQUESTED** [review]

File: cmd/kyma/alpha/uninstall/cmd.go (hunk 2 above)
Comment (tobiscr) [reply cmd/kyma/alpha/uninstall/cmd.go:112]: right.. let me add that

Comment (tobiscr) [reply cmd/kyma/alpha/uninstall/cmd.go:112]: fixed

File: cmd/kyma/alpha/uninstall/cmd.go [hunk 3]
```diff
@@ -92,15 +135,67 @@ func (cmd *command) Run() error {
...
+		if metadata.Version == "" {
+			getMetaStep.Failure()
+			return metadata, fmt.Errorf("No Kyma installation found")
+		}
+		getMetaStep.Successf("Kyma was installed from source '%s'", metadata.Version)
+	} else {
+		getMetaStep.Failure()
+	}
+	return metadata, err
+}
```
Comment (skhalash) [inline cmd/kyma/alpha/uninstall/cmd.go:191]: If we get more metadata-related logic, I'd consider factoring it out in smth like a `MetadataClient` to share with e.g. the version command.

**Review by skhalash — DISMISSED** [review]

**Review by suleymanakbas91 — DISMISSED** [review]

File: cmd/kyma/alpha/uninstall/cmd.go (hunk 3 above)
Comment (tobiscr) [reply cmd/kyma/alpha/uninstall/cmd.go]: Isn't the provider already a kind of client?

**Review by skhalash — APPROVED** [review]

**Review by suleymanakbas91 — APPROVED** [review]

**Review by alexandra-simeonova — APPROVED** [review]

**Pull request merged**